docs/mkdocs.yml

```diff
@@ -87,3 +88,12 @@ extra:
     - icon: social-discussion
       link: https://github.com/protectai/nbdefense/discussions
   generator: false
+
+extra_javascript:
+  - https://cdnjs.cloudflare.com/ajax/libs/timeago.js/4.0.2/timeago.min.js
+  - https://cdnjs.cloudflare.com/ajax/libs/lightgallery-js/1.4.1-beta.0/js/lightgallery.min.js
+  - assets/js/timeago_mkdocs.js
```

Comment:
I think you can remove these `assets/js/timeago_mkdocs.js` and `assets/js/lightgallery_mkdocs.js` files and items from here

Reply:
These files are needed to implement `lightgallery` and `timeago` to mkdocs.